resources/views/components/select/user-option.blade.php

```diff
@@ -6,9 +6,22 @@
     :readonly="$readonly"
     :option="$option"
 >
-    <div class="flex items-center gap-x-3">
-        <img src="{{ data_get($option, 'img', $img) }}" class="shrink-0 h-6 w-6 rounded-full">
+    <div>
+        @if ($subtitle)
```

Comment:
Instead of using the condition, try to simplify the HTML

```
<div class="flex items-center gap-x-3">
    <img src="{{ data_get($option, 'img', $img) }}" class="shrink-0 h-6 w-6 rounded-full">

    <div>
        <p>{!! $label !!}</p>

        @if ($subtitle)
            <span class="text-xs text-gray-400">{!! $subtitle !!}</span>
        @endif
    </div>
</div> 

```